modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/AbstractWFSStrategy.java

```diff
@@ -341,6 +341,10 @@ protected Map<String, String> buildGetFeatureParametersForGET(GetFeatureRequest
             map.put("MAXFEATURES", String.valueOf(request.getMaxFeatures()));
         }
 
+        if (request.getStartIndex() != null) {
```

Comment:
This covers the KVP requests, but not the POST ones that send an XML document.
See how maxFeatures is handled in StrictWFS_1_x_Strategy and StrictWFS_2_0_Strategy, and please add tests for those cases as well (they will have to check that the XML document representing the request contains the startIndex attribute).

Reply:
Added changes to bring the `startIndex` also on POST requests.
Thanks.